### PR TITLE
Move the learning of the vocabulary to the setup part

### PR DESCRIPTION
**Pull request by sgoycoechea**

### CODE REVIEW

File: scripts/train-mask-align/preprocess_europarl.sh [hunk 1]
```diff
@@ -2,16 +2,6 @@
 
 set -ex
 
-mkdir -p corpus-es
-cd corpus-es/
```
Comment (bryant1410) [inline scripts/train-mask-align/preprocess_europarl.sh:6]: The `cd corpus-es/` should be kept, right?

**Review by bryant1410 — CHANGES_REQUESTED** [review]
Looks good. Just one comment.

File: README.md [hunk 2]
```diff
@@ -35,7 +41,7 @@ Alternatively, follow these steps to train it yourself.
 
 ### Prepare the Corpus
 
-Run the following script to download the [Europarl Spanish-English parallel corpus](https://www.statmt.org/europarl/v7/es-en.tgz), do some preprocessing, learn the vocabulary, tokenize the sentences, and split the corpus:
+Run the following script to download the [Europarl Spanish-English parallel corpus](https://www.statmt.org/europarl/v7/es-en.tgz), do some preprocessing, tokenize the sentences, and split the corpus:
```
Comment (bryant1410) [inline README.md]: Oh, also the download part should go in the Setup now, tight?

File: README.md [hunk 3]
```diff
@@ -35,7 +41,7 @@ Alternatively, follow these steps to train it yourself.
```
Comment (bryant1410) [inline README.md:41]: BTW, do you also wanna change the pretrained model URL for the one under an asset in the only release from the repo?

File: scripts/train-mask-align/preprocess_europarl.sh (hunk 1 above)
Comment (sgoycoechea) [reply scripts/train-mask-align/preprocess_europarl.sh:6]: It's in the setup script, so if they are following the steps they should already be in corpus-es/

Should we add it here anyways?

File: README.md (hunk 2 above)
Comment (sgoycoechea) [reply README.md]: Yes, because the vocabulary is learned from that corpus.

File: scripts/train-mask-align/preprocess_europarl.sh (hunk 1 above)
Comment (bryant1410) [reply scripts/train-mask-align/preprocess_europarl.sh:6]: When you run a script, it doesn't leave your current fit changed if it changes internally. Unless you `source` it (or with `.`).

File: README.md (hunk 2 above)
Comment (bryant1410) [reply README.md]: I mean to move the text in the README to the setup part!

Comment (sgoycoechea) [reply README.md]: Ohh right.

File: README.md (hunk 3 above)
Comment (sgoycoechea) [reply README.md:41]: Done

**Review by bryant1410 — APPROVED** [review]

**Pull request merged**